RHIDP-9010: AEM-related changes (#1426)
redhat-developer/red-hat-developers-documentation-rhdh: modules/configuring-the-global-header/proc-displaying-preferred-username-in-profile-drop-down.adoc

You can display the preferred username in the global header profile drop-down list by configuring `spec.profile.displayName` in the user entity. When not configured, the application falls back to a `metadata.title`. If neither is configured, it defaults to a user-friendly name generated by the `useProfileInfo` hook.

.Procedure

> * When you configure `spec.profile.displayName`, use the following code:

Reviewer: ```suggestion
. To configure `spec.profile.displayName`, use the following code:
```

  memberOf: [janus-authors]
----

> * When you do not configure `spec.profile.displayname` but configure `metadata.title`, use the following code:

Reviewer: ```suggestion
. To configure `metadata.title` rather than `spec.profile.displayname`, use the following code:
```
